HerPortal.BusinessLogic/Models/LocalAuthorityData.cs

```diff
@@ -1,7 +1,14 @@
 ﻿namespace HerPortal.BusinessLogic.Models;
 
+/// <summary>
+///     This model is not a full model for Local Authority Data.
+///     It should be considered as a reference to the full data for a Local Authority.
+///     The full data (including an LAs relationship to a Consortium) can be found in the HUG2 Public Website codebase
+/// </summary>
```

Comment:
Not sure this makes sense in the context of this class?

Suppose it's worth mentioning somewhere here that we expect the mapping to match the other repo